src/openms/source/TRANSFORMATIONS/FEATUREFINDER/MultiplexClustering.cpp

```diff
@@ -73,13 +73,11 @@ namespace OpenMS
     double rt_min = exp_profile.getMinRT();
     double rt_max = exp_profile.getMaxRT();
     
-    // extend the grid by a small absolute margin
-    double mz_margin = 1e-10;
-    double rt_margin = 1e-10;
-    mz_min -= mz_margin; 
-    mz_max += mz_margin; 
-    rt_min -= rt_margin;
-    rt_max += rt_margin;
+    // extend the grid by a small machine-epsilon-dependent margin
+    mz_min -= 2 * mz_min * std::numeric_limits<double>::epsilon();
+    mz_max += 2 * mz_max * std::numeric_limits<double>::epsilon();
+    rt_min -= 2 * rt_min * std::numeric_limits<double>::epsilon();
+    rt_max += 2 * rt_max * std::numeric_limits<double>::epsilon();
```

Comment:
Use `fabs(rt_min)` and `fabs(rt_max)` here, since RTs can be negative (e.g., after alignment).